### DISCUSSION TITLE
[v5] ace make:migration command not found

### DISCUSSION THREAD

**Discussion by matscode** [adonisjs/core#2403] [Help]
I tried creating a migration file but what I got was `[ error ]  "make:migration" command not found` . 

I ran `node ace --help` and there is no command for migrations. But there is documentation for it.. Am I missing something?

**Comment by thetutlage** [comment]
Follow the setup docs here https://preview.adonisjs.com/guides/database/setup

**Reply by RockyScube** [reply]
this is not accessible.

**Reply by RomainLanz** [reply]
https://docs.adonisjs.com/guides/database/introduction
